pkg/controller/service/servicecontroller.go

```diff
@@ -378,7 +378,9 @@ func (s *ServiceController) createLoadBalancer(service *api.Service) error {
 		return err
 	}
 	name := s.loadBalancerName(service)
-	status, err := s.balancer.EnsureTCPLoadBalancer(name, s.zone.Region, net.ParseIP(service.Spec.LoadBalancerIP),
+	// getPortsForLB already verified that the protocol matches for all ports.
+	// The cloud provider will verify the protocol is supported
```

Comment:
please reword the comment:
1. don't use the name of the function
2. say something like: we only support a single protocol per service
3. say something like: not all cloud providers support all protocols, and the next step will return an error on unsupported protocol